### PR TITLE
Add submission instruction for openqa

### PR DESCRIPTION
**Pull request by insop**
- added submission file list

Screen Shot 2022-06-08 at 10 57 13 PM

Screen Shot 2022-06-08 at 10 57 20 PM

### CODE REVIEW

File: hw_openqa.ipynb [hunk 1]
```diff
@@ -65,7 +65,8 @@
     "    1. [Few-shot OpenQA [2 points]](#Few-shot-OpenQA-[2-points])\n",
     "    1. [Answer scoring [2 points]](#Answer-scoring-[2-points])\n",
     "    1. [Your original system [3 points]](#Your-original-system-[3-points])\n",
-    "1. [Bake-off [1 point]](#Bake-off-[1-point])"
+    "1. [Bake-off [1 point]](#Bake-off-[1-point])\n",
+    "1. [Submission Instruction](#Submission-Instruction)"
```
Comment (cgpotts) [inline hw_openqa.ipynb:69]: Please change this title to "Submission instructions" (note the capitalization and pluralization).

File: hw_openqa.ipynb [hunk 2]
```diff
@@ -2578,20 +2579,19 @@
...
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "\n"
+    "## Submission Instruction\n",
```
Comment (cgpotts) [inline hw_openqa.ipynb:2585]: "Submission instructions"

File: hw_openqa.ipynb [hunk 3]
```diff
@@ -2578,20 +2579,19 @@
...
+    " \n",
+    "Review and follow the [Homework and bake-off code: Formatting guide](hw_formatting_guide.ipynb).\n",
+    "Please **do not change the file name** as described below.\n",
```
Comment (cgpotts) [inline hw_openqa.ipynb:2588]: "filenames" instead of "file name".

**Review by cgpotts — CHANGES_REQUESTED** [review]
I requested a few changes to the wording.

File: hw_openqa.ipynb (hunk 1 above)
Comment (insop) [reply hw_openqa.ipynb]: Updated, PTAL.

File: hw_openqa.ipynb (hunk 2 above)
Comment (insop) [reply hw_openqa.ipynb]: Updated.

File: hw_openqa.ipynb (hunk 3 above)
Comment (insop) [reply hw_openqa.ipynb]: Updated.

**Pull request merged**